openedx/core/lib/xblock_builtin/xblock_discussion/static/js/mathjax_include.js

```diff
@@ -0,0 +1,36 @@
+var vendorScript;
```

Comment:
@andy-armstrong I changed the code to include this file in the JS discussion bundle: https://github.com/edx/edx-platform/pull/13483/commits/7d311c378542ad17b7676c10fa2834a6b307ee28.

As for refactoring `mathjax_include.html`: This is tricky because it is used on several different pages (and not just for discussion posts), with varying values of `disable_fast_preview` and `mathjax_mode`:

-   Wiki
    -   lms/templates/wiki/base.html
        -   `disable_fast_preview`: True
        -   `mathjax_mode`: "wiki"
    -   lms/templates/wiki/preview_inline.html
        -   `disable_fast_preview`: True
        -   `mathjax_mode`: "wiki"

-   Teams (via _js_body_dependencies.html)
    -   lms/djangoapps/teams/templates/teams/teams.html
        -   `disable_fast_preview`: True
        -   `mathjax_mode`: Undefined

-   Discussion (via _js_body_dependencies.html)
    -   lms/djangoapps/discussion/templates/discussion/discussion_profile_page.html
        -   `disable_fast_preview`: True
        -   `mathjax_mode`: Undefined
    -   lms/djangoapps/discussion/templates/discussion/discussion_board.html
        -   `disable_fast_preview`: False
        -   `mathjax_mode`: Undefined

-   Course
    -   lms/templates/courseware/courseware.html
        -   `disable_fast_preview`: False
        -   `mathjax_mode`: Undefined
    -   lms/templates/courseware/courseware-chromeless.html
        -   `disable_fast_preview`: False
        -   `mathjax_mode`: Undefined

-   "Static tab"
    -   lms/templates/courseware/static_tab.html
        -   `disable_fast_preview`: True
        -   `mathjax_mode`: Undefined

So what I did to create this file (`mathjax_include.js`) was to extract the (non-contiguous) regions of code that applied to the "Course" tab from `mathjax_include.html`.

To completely DRY out `mathjax_include.html`, it seems that we'd need to create multiple JS files from the different sections of that file, and then include them based on the values of `disable_fast_preview` and `mathjax_mode`. An alternative approach would be to remove `mathjax_include.html` and include the individual JS files directly in each of the templates (again based on the values of `disable_fast_preview` and `mathjax_mode`). In any case, the code in the individual JS files would need to be reusable so that this file (`mathjax_include.js`) could combine it as it does now.

Since this would impact many important parts of the system, and goes beyond enabling `DiscussionXBlock` to load assets it depends on, I think it would make sense to handle it in a separate PR.